Cargo.toml

```diff
@@ -7,13 +7,12 @@ description = "Random utf8 utility"
 keywords = ["rand", "random", "utf8"]
 categories = ["algorithms", "cryptography"]
 license = "MIT/Apache-2.0"
-readme = "README.md"
 documentation = "https://docs.rs/rand-utf8"
-repository = "https://github.com/neonphog/rand-utf8"
+repository = "https://github.com/holochain/rand-utf8"
```

Comment:
Updated to point back to here, which is where the code will be when we publish a new version now